locales/it.yml

```diff
@@ -0,0 +1,12 @@
+---
+it:
+  header:
+    title: "Berlin Code of Conduct"
+    sub-title: "Un Codice di Condotta per tutti gli User Groups e conferenze"
+  supporters:
+    title: "Aderenti"
+    sub-title: "Gli user groups e conferenze che supportano il Berlin Code of Conduct"
```

Comment:
Foreign words don't get the plural form, usually.
http://forum.accademiadellacrusca.it/forum_5/interventi/1685.shtml.html 

> la regola vale per tutte le parole straniere: invariabili se entrate nell'uso comune, col plurale d'origine se sentite come esotismi.

I am not sure this can be considered an "exotic" word.